pkg/util/config/config.go

```diff
@@ -45,6 +45,7 @@ type Resources struct {
 	MinCPULimit             string              `name:"min_cpu_limit" default:"250m"`
 	MinMemoryLimit          string              `name:"min_memory_limit" default:"250Mi"`
 	PodEnvironmentConfigMap spec.NamespacedName `name:"pod_environment_configmap"`
+	PodEnvironmentSecret    string              `name:"pod_environment_secret"`
```

Comment:
why isn't this of type `spec.NamespacedName` too?

Reply:
To avoid exposing the contents of the secrets, main reason for this PR, they are not copied (as with the existing PodEnvironmentConfigMap ) but referenced. This means only a secret in the pods namespace can be used here (see https://kubernetes.io/docs/concepts/configuration/secret/#restrictions)

Reply:
I see, it's also documented. Fine for me then.